[2024-01-27T02:26:54.532+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_dag.extract_task manual__2024-01-27T02:26:51.912608+00:00 [queued]>
[2024-01-27T02:26:54.538+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_dag.extract_task manual__2024-01-27T02:26:51.912608+00:00 [queued]>
[2024-01-27T02:26:54.538+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-01-27T02:26:54.570+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): extract_task> on 2024-01-27 02:26:51.912608+00:00
[2024-01-27T02:26:54.574+0000] {standard_task_runner.py:57} INFO - Started process 10768 to run task
[2024-01-27T02:26:54.576+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'extract_dag', 'extract_task', 'manual__2024-01-27T02:26:51.912608+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/extract_nsdq.py', '--cfg-path', '/tmp/tmpu6m8n1gk']
[2024-01-27T02:26:54.577+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask extract_task
[2024-01-27T02:26:54.624+0000] {task_command.py:410} INFO - Running <TaskInstance: extract_dag.extract_task manual__2024-01-27T02:26:51.912608+00:00 [running]> on host codespaces-ca3784
[2024-01-27T02:26:54.709+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='extract_dag' AIRFLOW_CTX_TASK_ID='extract_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-27T02:26:51.912608+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-27T02:26:51.912608+00:00'
[2024-01-27T02:26:54.710+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-27T02:26:54.711+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'wget -c https://data.nasdaq.com/api/v3/datasets/WIKI/AAPL.json -O /workspaces/hands-on-introduction-data-engineering-4395021/lab/extract-nsdq.csv']
[2024-01-27T02:26:54.717+0000] {subprocess.py:86} INFO - Output:
[2024-01-27T02:26:54.786+0000] {subprocess.py:93} INFO - --2024-01-27 02:26:54--  https://data.nasdaq.com/api/v3/datasets/WIKI/AAPL.json
[2024-01-27T02:26:54.838+0000] {subprocess.py:93} INFO - Resolving data.nasdaq.com (data.nasdaq.com)... 45.60.150.18
[2024-01-27T02:26:54.847+0000] {subprocess.py:93} INFO - Connecting to data.nasdaq.com (data.nasdaq.com)|45.60.150.18|:443... connected.
[2024-01-27T02:26:55.096+0000] {subprocess.py:93} INFO - HTTP request sent, awaiting response... 429 Too Many Requests
[2024-01-27T02:26:55.096+0000] {subprocess.py:93} INFO - 2024-01-27 02:26:55 ERROR 429: Too Many Requests.
[2024-01-27T02:26:55.097+0000] {subprocess.py:93} INFO - 
[2024-01-27T02:26:55.097+0000] {subprocess.py:97} INFO - Command exited with return code 8
[2024-01-27T02:26:55.102+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/python/3.10.13/lib/python3.10/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 8.
[2024-01-27T02:26:55.104+0000] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=extract_dag, task_id=extract_task, execution_date=20240127T022651, start_date=20240127T022654, end_date=20240127T022655
[2024-01-27T02:26:55.140+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 3 for task extract_task (Bash command failed. The command returned a non-zero exit code 8.; 10768)
[2024-01-27T02:26:55.149+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2024-01-27T02:26:55.162+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
